apps/teamsjs-cc-app/tabs/src/components/PagesTab.tsx

```diff
@@ -0,0 +1,106 @@
+                    </Fluent.Flex>
+                </Fluent.Segment>
+                <Fluent.Segment className="ui-pagessegment">
+                    <Fluent.Header content="Return Focus To Search Box" as="h2" />
+                    <Fluent.Flex gap="gap.small" vAlign="center">
+                        <Fluent.Text className="ui-pagestext" content="Return focus to the host. Will move focus forward or backward based on where the application container falls in the F6/tab order in the host. On mobile hosts or hosts where there is no keyboard interaction or UI notion of 'focus' this function has no effect and will be a no-op when called." />
+                    </Fluent.Flex>
+                    <Fluent.Flex space="between"><Fluent.Button onClick={async () => {
+                        pages.returnFocus(true);
+                    }}>
+                        Click me to Return focus to search box
+                    </Fluent.Button></Fluent.Flex>
+                </Fluent.Segment>
+                <Fluent.Segment className="ui-pagessegment">
+                    <Fluent.Header content="Set Current Frame to 'Terms of Use'" as="h2" />
+                    <Fluent.Flex gap="gap.small" vAlign="center">
+                        <Fluent.Text className="ui-pagestext" content="Sets/Updates the current frame with new information." />
+                    </Fluent.Flex>
+                    <Fluent.Flex space="between"><Fluent.Button onClick={() => {
+                        const baseUrl = `https://${window.location.hostname}:${window.location.port}`;
+                        pages.setCurrentFrame({
+                            contentUrl: `${baseUrl}/#/termsofuse`,
+                            websiteUrl: `${baseUrl}/#/termsofuse`
+                        });
+                        showText(true);
+                    }}>
+                        Set Current Frame to 'Terms of Use'
+                    </Fluent.Button>
+                    </Fluent.Flex>
+                    {isText &&
+                        <Fluent.Flex gap="gap.small" vAlign="center">
+                            <Fluent.Text className="ui-pagestext" content="Please reload tab to see the effect." />
+                        </Fluent.Flex>
+                    }
+                </Fluent.Segment>
+                <Fluent.Segment className="ui-pagessegment">
+                    <Fluent.Header content="Share Deep Link" as="h2" />
+                    <Fluent.Flex gap="gap.small" vAlign="center">
+                        <Fluent.Text className="ui-pagestext" content="Shares a deep link that a user can use to navigate back to a specific state in this page. Please note that this method does yet work on mobile hosts." />
+                    </Fluent.Flex>
+                    <Fluent.Flex space="between"><Fluent.Button onClick={async () => {
+                        pages.shareDeepLink({
+                            subPageId: "72c73d2e-a890-4580-9c68-513c8cb6efcd",
```

Comment:
Another possible candidate for being made a const if this subpageid is references in multiple places